includes/Traits/Controls.php

```diff
@@ -1659,13 +1669,30 @@ public static function layout($wb)
 	        }
 
 	        foreach ( $post_types as $post_type => $post_taxonomies ) {
+		        $options = [];
+                $default = '';
+
+                if( isset( $post_types_tax[ $post_type ] ) ){
```

Comment:
Let's use proper spacing for if block (1 space before and after the parenthesis).
example: `if () {}`